Casks/keyman.rb

```diff
@@ -0,0 +1,37 @@
+cask "keyman" do
+  version "14.0.280"
+  sha256 "8465f512b23900f8b6527cf4a2f21c550d526bac7135607b3abfd1609608b91a"
+
+  url "https://downloads.keyman.com/mac/stable/#{version}/keyman-#{version}.dmg"
+  name "Keyman"
+  desc "Reconfigures keyboard to type in another language"
+  homepage "https://keyman.com/"
+
+  livecheck do
+    url "https://downloads.keyman.com/mac/stable"
```

Comment:
```suggestion
    url "https://downloads.keyman.com/api/version/mac"
    strategy: page_match
```
and update the regex for the response

```
{
    "mac": {
        "alpha": "15.0.107",
        "beta": "14.0.268",
        "stable": "14.0.280"
    }
}
```

Reply:
Yes, the regex for that could be
```ruby
regex(%r{"stable": "(\d+(?:\.\d+)+)",?}i)
```
But I'm not sure if this is a good idea to @bevanjkay.

Reply:
@darcywong00 Is there an instance where the stable archive with the downloads folder would be ahead of the current release? Otherwise, as per [livecheck URL guidelines](https://docs.brew.sh/Brew-Livecheck#url-guidelines) we should check the same location as the stable archive.

Reply:
According to the [API doc](https://github.com/keymanapp/keyman/wiki/downloads.keyman.com-version-API#api-use-notice), this endpoint may "change without notice". This could be a reason not to use it though.

Reply:
> Otherwise, as per livecheck URL guidelines we should check the same location as the stable archive.

Give all the above, go ahead and use your original url. My apologies.

Reply:
Well, then I was too eager...